src/stories/Quiz/argTypes.ts

```diff
@@ -82,6 +82,20 @@ export const argTypes = {
       },
     },
   },
+  'callbacks.onEmailResults': {
+    description: 'Callback function to be called when emailing results./',
```

Comment:
I believe the `/` at end of line here is extra

Reply:
030dc0f3119ecfe746210561c897aa72e60855db